File: test/llvm_test_code/linear_constant/global_09.cpp
int g1 = 1;

int foo(int a) {
  a += g1;
  return a;
}

int bar(int b) {
  g1 += 1;
  return b + 1;
}

int baz(int c) {
  return c + 3;
}

int main() {
	g1 += 1;
  int i = 0;
  i = foo(10);
  i = bar(3);
  i = baz(39);
	return 0;
}

File: test/llvm_swift_test_code/linear_constant/global_09.swift
var g1 = 1

@main
struct MyMain {

  static func baz (_ c: Int)-> Int {
    return c + 3
  }

  static func bar (_ b: Int)-> Int {
    g1 += 1
    return b + 1
  }

  static func foo (_ a: Int)-> Int {
    var x = a
    x+=g1
    return x
  }

    static func main() {
      g1 += 1
      var i = 0
      i = foo(10)
      i = bar(3)
      i = baz(39)
    }
}
